tests/Functional/EnvCommandsTest.php

```diff
@@ -261,7 +261,7 @@ public function testListCommand()
     public function testViewCommand()
     {
         $url = $this->terminus(sprintf('env:view %s --print', $this->getSiteEnv()));
-        $expectedUrl = sprintf('https://%s-%s.pantheonsite.io/', $this->getMdEnv(), $this->getSiteName());
```

Comment:
Any specific reason to make this change?

Reply:
No, I will revert it. I was trying to find out the issue with the build failures.